core-errors/src/lib.rs

```diff
@@ -71,6 +70,28 @@ pub enum MessageError {
     /// a single execution.
     #[display(fmt = "Duplicated program initialization message")]
     DuplicateInit,
+
+    #[display(fmt = "User has no enough gas")]
+    NotEnoughGas,
+
+    #[display(
```

Comment:
Pls add description:
/// Existential deposit is a minimal amount of funds on a balance that can be considered and added in DB. Everything less than existential deposit but greater than 0 is not considered as available balance and not saved in DB.  Value between 0 and existential deposit cannot be sent in message.